### DISCUSSION TITLE
Registration Hollie French

### DISCUSSION THREAD

**Discussion by HFrench20** [IBAHCM/RPiR#283] [Archive]
Please can I be added to the discussion for the RPiR course. I am a PGR working through it a bit late, but it will be useful to see the history.

**Comment by soniamitchell** [comment]
Done 👍 You should get an invite by email, or possibly if you go here https://github.com/IBAHCM

**Comment by HFrench20** [comment]
Hi Sonia,

Thanks yes that seems to have all gone through fine 😊

Best wishes,
Hollie
